### PR TITLE
Add kdenlive

### PR DESCRIPTION
**Pull request by dmlemos**
After making all changes to a cask, verify:

- [X] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [X] `brew audit --cask <cask>` is error-free.
- [X] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [X] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [X] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [X] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [X] `brew audit --new-cask <cask>` worked successfully.
- [X] `brew install --cask <cask>` worked successfully.
- [X] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/kdenlive.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  url "https://cdn.download.kde.org/stable/kdenlive/#{version.major_minor}/macOS/kdenlive-#{version}.dmg",
+      verified: "cdn.download.kde.org/stable/kdenlive/"
+  name "Kdenlive"
+  desc "Free and Open Source Video Editor"
+  homepage "https://kdenlive.org/"
+
+  livecheck do
+    url "https://kdenlive.org/en/download/"
+    regex(%r{href=['"]?https://download.kde.org/stable/kdenlive/\d+(?:\.\d+)+/macOS/kdenlive-(\d+(?:\.\d+)+(?:\.\d+)+).dmg['"]?}i)
```
[the remark this answers is not in the document]
Comment (dmlemos) [reply Casks/kdenlive.rb:13]: yeah. The reason I did the more complicated version, was to force look for the stable version.

What about this?
```ruby
regex(%r{stable/kdenlive/\d+(?:\.\d+)+/macOS/kdenlive-(\d+(?:\.\d+)+(?:\.\d+)+).dmg['"]?}i)
```

[the remark this answers is not in the document]
Comment (SMillerDev) [reply Casks/kdenlive.rb:13]: Yeah, if that works it looks okay to me. Just wanted to avoid the whole domain in there

**Comment by dmlemos** [comment]
Rebased with the discussed change.

**Review by SMillerDev — APPROVED** [review]
LGTM, thanks @dmlemos!

**Pull request merged**